multisig/proposals/opsepolia/vip-002/index.ts

```diff
@@ -0,0 +1,303 @@
+import { parseUnits } from "ethers/lib/utils";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { makeProposal } from "src/utils";
+
+const { opsepolia } = NETWORK_ADDRESSES;
+
+const ACM = "0x1652E12C8ABE2f0D84466F0fc1fA4286491B3BC1";
+
+export const XVS_BRIDGE_ADMIN_PROXY = "0x6bBcB95eCF9BEc9AE91d5Ad227783e3913145321";
+export const XVS = "0x789482e37218f9b26d8D9115E356462fA9A37116";
+export const XVS_BRIDGE_DEST = "0x79a36dc9a43D05Db4747c59c02F48ed500e47dF1";
+
+export const BNB_TESTNET_ENDPOINT_ID = 10102;
+export const SEPOLIA_ENDPOINT_ID = 10161;
+export const OPBNB_TESTNET_ENDPOINT_ID = 10202;
+export const ARBITRUM_SEPOLIA_ENDPOINT_ID = 10231;
+export const ZKSYNC_SEPOLIA_ENDPOINT_ID = 10248;
+
+export const OPBNB_TESTNET_TRUSTED_REMOTE = "0xa03205bc635a772e533e7be36b5701e331a70ea3";
+export const SEPOLIA_TRUSTED_REMOTE = "0xc340b7d3406502f43dc11a988e4ec5bbe536e642";
+export const BNB_TESTNET_TRUSTED_REMOTE = "0x0e132cd94fd70298b747d2b4d977db8d086e5fd0";
+export const ARBITRUM_SEPOLIA_TRUSTED_REMOTE = "0xfdc5cec63fd167da46cf006585b30d03b104efd4";
+export const ZKSYNC_SEPOLIA_TRUSTED_REMOTE = "0x760461ccb2508caaa2ece0c28af3a4707b853043";
+
+export const SINGLE_RECEIVE_LIMIT = parseUnits("10200", 18);
+export const MAX_DAILY_RECEIVE_LIMIT = parseUnits("51000", 18);
+export const MIN_DST_GAS = "300000";
+export const SINGLE_SEND_LIMIT = parseUnits("10000", 18);
+export const MAX_DAILY_SEND_LIMIT = parseUnits("50000", 18);
+
+export const XVS_MINT_LIMIT = parseUnits("500000", 18);
+
+const vip002 = () => {
+  return makeProposal([
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "setSendVersion(uint16)", opsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "setReceiveVersion(uint16)", opsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "forceResumeReceive(uint16,bytes)", opsepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [XVS_BRIDGE_ADMIN_PROXY, "setOracle(address)", opsepolia.NORMAL_TIMELOCK],
```

Comment:
```suggestion
      params: [XVS_BRIDGE_ADMIN_PROXY, "setOracle(address)", opsepolia.GUARDIAN],
```
Let's modify each instance so that after multichain governance, this VIP can still be successfully simulated

Reply:
Updated, 8a144ff308eb2a83393043ca965a180418f91bda